nservicebus/hosting/nservicebus-host/troubleshooting.md

```diff
@@ -34,17 +34,21 @@ Verify that:
 1. Assembly binding redirects in the `NServiceBus.Host.exe.config` are correct and match with any entries generated in the `App.config` file.
 2. All files in the deployment (sub)folder are correct and no old files are present from previous deployments.
 
-## Service fails to start due to reaching the timeout period
+## Windows Service fails to start due to reaching the timeout period
 
-Sometimes when the server hosting a microservice that uses NServiceBus is started, the following exception is seen in windows event viewer.
-> A timeout was reached (30000 milliseconds) while waiting for the XYZ service to connect.
-> The XYZ service failed to start due to the following error:
-> The service did not respond to the start or control request in a timely fashion
+Sometimes, a Windows service that is set to start automatically will fail to start due to a dependency not yet being available before it times out. The following exception can be seen in the Windows Event Viewer:

```

Comment:
Now that I reviewing it again I think this whole docs should just point to https://docs.particular.net/nservicebus/hosting/windows-service which already has documentation on sc.exe